op-chain-ops/cmd/celo-migrate/main.go

```diff
@@ -213,58 +197,118 @@ func main() {
 	if err := app.Run(os.Args); err != nil {
 		log.Crit("error in migration", "err", err)
 	}
-	log.Info("Finished migration successfully!")
 }
 
-func runBlockMigration(opts blockMigrationOptions) error {
+func runFullMigration(opts fullMigrationOptions) error {
+	defer timer("full migration")()
+
+	log.Info("Full Migration Started", "oldDBPath", opts.oldDBPath, "newDBPath", opts.newDBPath)
+
+	var err error
+	var numAncients uint64
+	var extraAncientNumHashes []*rawdb.NumberHash
+
+	if extraAncientNumHashes, numAncients, err = runPreMigration(opts.preMigrationOptions); err != nil {
```

Comment:
I don't really understand the naming here for `extraAncientNumHashes`, it's returned along with `numAncients` which is the number of ancients (this naming makes sense to me) but then `extraAncientNumHashes` is a list of what I would call dangling block hashes, blocks that somehow did not get cleared from the database when moving blocks to the ancients. But the naming makes it sound like it's the number of extra ancient hashes, which it is not.

So my suggestion would be to call it `danglingBlocks`, `lostBlocks`, `abandonedBlocks` or something similar. And rename the function and any other references accordingly.